infrastructure/ecs_task/outputs.tf

```diff
@@ -19,7 +19,7 @@ output "results_bucket_id" {
 }
 
 output "task_queue_consumer" {
```

Comment:
This output is now identical to trigger_role_name. Please remove one or other of them